### ISSUE TITLE
lazy-loading.md '@/lang' change to '@/lang/en'

### ISSUE THREAD

**Issue by duyajun** [kazupon/vue-i18n#292]
### vue & vue-i18n version
ex: 2.1.0, 5.0

### Reproduction Link

gitbook/en/lazy-loading.md

### What is Expected?
import messages from '@/lang'
'@/lang' change to '@/lang/en', director cannot be imported.

**Comment by VictorCazanave** [comment]
@kazupon then maybe the project directory should be updated adding `index.js` between `en.js` and `it.js`?

Actually I'm a bit confused with the example, because I don't understand what `messages` and `msgs` contain.
Could you clarify it please?

**Comment by rwfresh** [comment]
I have to agree. Spent a few hours trying to determine what the structure of the language files should be and now that I found this link I confirmed I need an index.js file but not entirely sure what it should look like. Maybe I am a bit slow but including a short overview of what the language files and the index.js file should look like would be really helpful. Thanks!
